discover/deployment/articles/04-securing-liferay/15-oauth2/02-authorizing-access.markdown

```diff
@@ -12,45 +12,89 @@ applications. The URL for this requires the following request parameters:
 
 - `response_type` 
 - `client_id` 
-- `client_secret` (optional)
+- `client_secret`
 
 To construct a URL for this authorization, therefore, follow this pattern: 
 
-    https://[hostname]/o/oauth2/authorize?response_type=code&client_id=[client ID]
+    https://[hostname]/o/oauth2/authorize?response_type=code&client_id=[client ID]&client_secret=[client secret]
```

Comment:
@stian-sigvartsen ... is the client_secret mandatory in this request? I would say it SHOULD NOT go in this request, should it?

Reply:
Sorry @sez11a , Carlos is right. I'll amend